cmd/eh-frame/main.go

```diff
@@ -64,7 +64,7 @@ func main() {
 	}
 
 	if flags.Final {
-		ut, arch, _, err := unwind.GenerateCompactUnwindTable(file)
+		ut, arch, _, err := unwind.GenerateCompactUnwindTable(file, logger, prometheus.NewCounter(prometheus.CounterOpts{}))
```

Comment:
Rather than passing a no-op counter, I would prefer to pass a no-op registerer.

I think we need a new type `CompactUnwindGenerator` and that can hold on to logger and register and crate it's own metrics.